src/peft/tuners/lora.py

```diff
@@ -1167,3 +1172,58 @@ def forward(self, x: torch.Tensor):
                         )
                     result += output
                 return result
+
+
+# Ugly solution
```

Comment:
What do you mean by that? :)

Also, can we not theoretically remove the check and always define `QuantLinear`? the class definition doesn't seem to have a dependency on GPTQ.

Reply:
Yeah we can remove it :)